packages/babel-helpers/src/helpers/objectSpread2.ts

```diff
@@ -0,0 +1,65 @@
+/* @minVersion 7.5.0 */
+
+// @ts-expect-error Migrate in another PR
+import defineProperty from "./defineProperty.ts";
+
+// This function is different to "Reflect.ownKeys". The enumerableOnly
+// filters on symbol properties only. Returned string properties are always
+// enumerable. It is good to use in objectSpread.
+
+function ownKeys(
+  object: object,
+  enumerableOnly?: boolean | undefined,
+): PropertyKey[] {
+  var keys: PropertyKey[] = Object.keys(object);
```

Comment:
This can be narrowed down to `(string | symbol)[]`. The `PropertyKey` is supposed to type the valid non-computed key syntax in the object literal, which also contains `number`, however, `Object.keys()` will always return `string`.

Reply:
Actually, `Object.keys` only returns strings and not symbols. TypeScript's build-in definition of `Object.keys` is fine.

Reply:
Oh I see, we later push symbols.